schema.sql

```diff
@@ -43,3 +43,28 @@ ALTER TABLE animals
   ADD CONSTRAINT fk_owner
   FOREIGN KEY (owner_id)
   REFERENCES owners (id);
+
+-- Creating new table vets
+CREATE TABLE vets (
+  id SERIAL PRIMARY KEY NOT NULL,
+  name VARCHAR(100) NOT NULL, 
+  age INT,
+  date_of_graduation DATE NOT NULL 
+)
+
+--Specialization table
+CREATE TABLE specializations (
+    species_id INT NOT NULL,
+    vet_id INT NOT NULL,
+    FOREIGN KEY (species_id) REFERENCES species (id),
+    FOREIGN KEY (vet_id) REFERENCES vets (id) 
+);
+
+--Visit table
+CREATE TABLE visits (
+    animal_id INT NOT NULL,
+    vet_id INT NOT NULL,
+    date_of_visit DATE,
+    FOREIGN KEY (animal_id) REFERENCES animals (id),
+    FOREIGN KEY (vet_id) REFERENCES vets (id) 
```

Comment:
![image](https://user-images.githubusercontent.com/65068771/159695549-018598c9-07b0-4c6e-8702-493303510b4d.png)

- [ ] Kindly note that these two tables :point_up:  are violating the unique `PRIMARY KEY` in the table. you are defining two primary keys in these tables.

```sql
-- Owners table 
CREATE TABLE owners (
    id SERIAL PRIMARY KEY NOT NULL,
    full_name varchar(100),
    age INT
);
--or
CREATE TABLE owners (
    id SERIAL NOT NULL,
    full_name varchar(100),
    age INT,
   PRIMARY KEY(id)
);
 

-- Species table 
CREATE TABLE species (
    id SERIAL PRIMARY KEY NOT NULL,
    name varchar(100)
);
--or 
CREATE TABLE species (
    id SERIAL NOT NULL,
    name varchar(100),
    PRIMARY KEY(id)
);
```

This is found error :point_down: 

![image](https://user-images.githubusercontent.com/65068771/159697687-bf199f75-8b2e-4c33-8050-60d96e3521dc.png)